Fixes couriergraph formattting:
 1. Bring the xpoints back to 540, in line with mailgraph.
 2. Align graph text and numbers again.

diff -urN couriergraph-0.25/couriergraph.cgi couriergraph-0.25+cosmetics/couriergraph.cgi
--- couriergraph-0.25/couriergraph.cgi	2007-02-07 12:50:42.000000000 +1030
+++ couriergraph-0.25+cosmetics/couriergraph.cgi	2007-02-07 12:53:22.000000000 +1030
@@ -14,7 +14,7 @@
 
 my $host = (POSIX::uname())[1];
 my $scriptname = 'couriergraph.cgi';
-my $xpoints = 800;
+my $xpoints = 540;
 my $points_per_sample = 3;
 my $ypoints = 160;
 my $ypoints_err = 96;
@@ -80,24 +80,24 @@
 		"CDEF:rmimapd_ssl_login=mimapd_ssl_login,60,*",
 	        "CDEF:vimapd_ssl_login=imapd_ssl_login,UN,0,imapd_ssl_login,IF,$range,*",
   	 	    
-		'LINE:rpop3d_login#DD0000:pop3',
-		'GPRINT:vpop3d_login:AVERAGE:total\: %.0lf logins',
-		'GPRINT:rmpop3d_login:MAX:max\: %.0lf logins/min\l',
+		'LINE:rpop3d_login#DD0000:pop3    ',
+		'GPRINT:vpop3d_login:AVERAGE:total\: %8.0lf logins',
+		'GPRINT:rmpop3d_login:MAX:max\: %4.0lf logins/min\l',
 		'HRULE:0#000000',
 	    
 		'AREA:rpop3d_ssl_login#770000:pop3/ssl:STACK',
-		'GPRINT:vpop3d_ssl_login:AVERAGE:total\: %.0lf logins',
-		'GPRINT:rmpop3d_ssl_login:MAX:max\: %.0lf logins/min\l',
+		'GPRINT:vpop3d_ssl_login:AVERAGE:total\: %8.0lf logins',
+		'GPRINT:rmpop3d_ssl_login:MAX:max\: %4.0lf logins/min\l',
 		'HRULE:0#000000',
 
-	        'LINE:rimapd_login#00DD00:imap',
-	        'GPRINT:vimapd_login:AVERAGE:total\: %.0lf logins',
-		'GPRINT:rmimapd_login:MAX:max\: %.0lf logins/min\l',
+	        'LINE:rimapd_login#00DD00:imap    ',
+	        'GPRINT:vimapd_login:AVERAGE:total\: %8.0lf logins',
+		'GPRINT:rmimapd_login:MAX:max\: %4.0lf logins/min\l',
 		'HRULE:0#000000',
 	    
 		'AREA:rimapd_ssl_login#007700:imap/ssl:STACK',
-		'GPRINT:vimapd_ssl_login:AVERAGE:total\: %.0lf logins',
-		'GPRINT:rmimapd_ssl_login:MAX:max\: %.0lf logins/min\l',
+		'GPRINT:vimapd_ssl_login:AVERAGE:total\: %8.0lf logins',
+		'GPRINT:rmimapd_ssl_login:MAX:max\: %4.0lf logins/min\l',
 
 		'COMMENT:\s',
 		'COMMENT:['.$date.']\r',
